Sources/RadioButton.swift

```diff
@@ -11,6 +11,13 @@ import UIKit
     @IBInspectable open var isSelected: Bool = false {
         didSet {
             selectedCenterView.isHidden = !isSelected
+            
+            if let selectedRingColor = selectedRingColor, isSelected {
+                layer.borderColor = selectedRingColor.cgColor
+            } else if let normalRingColor = normalRingColor, !isSelected {
```

Comment:
I don't see the need for `normalRingColor` - it duplicates the same functionality of `tintColor`. You can use `actualTintColor` instead.